pandas/core/indexing.py

```diff
@@ -842,8 +842,14 @@ def _getitem_nested_tuple(self, tup: Tuple):
             if self.name != "loc":
                 # This should never be reached, but lets be explicit about it
                 raise ValueError("Too many indices")
-            with suppress(IndexingError):
-                return self._handle_lowerdim_multi_index_axis0(tup)
+            if isinstance(self.obj, ABCSeries) or not any(
```

Comment:
`self.ndim == 1` is faster than this isinstance check (41 ns vs 297 ns, so not a huge deal either way)

Reply:
Is better to read too :)